stacks/api/bucket/src/bucket-data.controller.ts

```diff
@@ -237,16 +269,41 @@ export class BucketDataController {
     return document;
   }
 
+  /**
+   * Adds a new document into the bucket. Keep in mind that the document in the body has to match schema of the bucket.
+   * @param bucketId Identifier of the bucket.
+   * @body
+   * ##### When the bucket has translated property body looks like below
+   * ```json
+   * {
+   *    "name": "Lucifer",
+   *    "age": 35,
+   *    "famous": true,
+   *    "role": "Actor"
+   * }
+   * ```
+   * ##### Otherwise, when there is a translated property, for instance the `name` property has to be like below
+   * ```json
+   * {
+   *    "name": {
+   *      "tr_TR": "Lusifır",
```

Comment:
Probably we can find a better and simple translation example. Maybe a book entry?

Reply:
Yes, that makes sense. Do you have anything in mind?  
It would be helpful if you can suggest a change